react_11/11-targil/guessingGame/src/main.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { useState } from 'react';
+
+const App = () => {
+  
+  const [target] = useState(Math.floor(Math.random() * 1000));
```

Comment:
If you only use `state` to set a variable, there is no actual need for it, you can simply use `const`. However, here you need to get a new number on a successful guess, so you do need a setter.